content/webapp/pages/exhibition-guide.tsx

```diff
@@ -22,6 +22,23 @@ import styled from 'styled-components';
 import { exhibitionGuidesLinks } from '@weco/common/views/components/Header/Header';
 import AudioPlayer from '@weco/common/views/components/AudioPlayer/AudioPlayer';
 import VideoEmbed from '@weco/common/views/components/VideoEmbed/VideoEmbed';
+import ButtonSolidLink from '@weco/common/views/components/ButtonSolidLink/ButtonSolidLink';
+
+type StopProps = {
+  width: string;
```

Comment:
I think we normally pass numeric things into styled components as numbers and add the e.g. `px` manually. Don't think it matters much, but nicer if e.g. we ever wanted to do maths with it